airbyte-integrations/connectors/source-mailchimp/integration_tests/standard_source_test.py

```diff
@@ -0,0 +1,51 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+import pkgutil
+
+from airbyte_protocol import AirbyteCatalog, ConnectorSpecification
+from base_python_test import StandardSourceTestIface
+
+
+class SourceMailchimpStandardTest(StandardSourceTestIface):
```

Comment:
Sorry, I realized that we did not document what you are supposed to do here. We will try to add documentation today. For now here is a brief explanation. 

We run a "standard" test suite for all integrations. These are pre-written tests. The idea is that all integrations should be able to pass these tests to prove that they have some basic functionality. In this case, the tests are already written, but you need to provide them some inputs so that they know how to interact with the integrations. This class defines those inputs.

* It looks like you are really close on this. The only thing that looks like it is missing is the `get_config`. This should pull a valid configuration (api_key and username) for the integration. Essentially it should just return the contents `airbyte-integrations/connectors/source-mailchimp/sample_files/test_config.json`. With that information the tests should have enough information to run.
* You can run these tests with the following command: `./gradlew :airbyte-integrations:connectors:source-mailchimp:standardSourceTestPython`. If you get failures and need to look at the actual tests to see what is wrong those test can be found [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/TestSource.java).